src/mongocxx/test/spec/change_stream.cpp

```diff
@@ -172,11 +186,11 @@ TEST_CASE("Change stream spec tests", "[change_stream_spec]") {
 
     client client{uri{}};
     if (!test_util::is_replica_set(client)) {
-        WARN("Skipping - not a replica set");
+        UNSCOPED_INFO("Skipping - not a replica set");
```

Comment:
I tried to reduce the number of warnings. Displaying everything as a warning made it difficult to debug.

Reply:
Actually, I checked this out locally. Running the test runner with `-s` shows the output of `INFO` and `UNSCOPED_INFO` logs.